rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -85,6 +87,13 @@ def ensure_global(name, default):
         # TIP: if using the sphinx-bootstrap-theme, you need
         # "treeViewIsBootstrap": True,
         "exhaleExecutesDoxygen": False,
+        # Maps markdown files to the "md" lexer, and not the "markdown" lexer
+        # Pygments registers "md" as a valid markdown lexer, and not "markdown"
+        "lexerMapping": {{r".*\.(md|markdown)$": "md",}},
+        # This mapping will work when `exhale` supports `:doxygenpage:` directives
+        # Check https://github.com/svenevs/exhale/issues/111
+        "customSpecificationsMapping": utils.makeCustomSpecificationsMapping(
+        lambda kind: [":project:", ":path:", ":content-only:"] if kind == "page" else []),
```

Comment:
Can you indent here, it looks strange to have these two lines be indented to the same level.

Reply:
Indented in 4768a1b.

FYI, I've also commented out the actual specification remapping because as it stands, this remapping causes unexpected behavior for any other directive (such as `:doxygenclass:`) not explicitly listed in the remapping. This can be addressed once support for `:doxygenpage:` directives is formally introduced into `exhale`.

If needed, I can just remove the commented lines out instead. In any case, those statements shouldn't execute now.

Reply:
Ok